packages/core/src/schema/platforms/fragments/templateAndroid.ts

```diff
@@ -25,7 +25,7 @@ Injects / Overrides values in res/values files of generated android based projec
 > IMPORTANT: always ensure that your object contains \`tag\` and \`name\` to target correct tag to merge into
  `);
 // We using interfaces to reduce the size of d.ts files (zod + types in d.ts files are huge)
-export interface ConfigTemplateAndroidResources extends z.infer<typeof zodAndroidResources> {}
+export type ConfigTemplateAndroidResources = z.infer<typeof zodAndroidResources>
```

Comment:
@aurimasmi specific reason to switch back to type here? using interfaces here is done on purpose to avoid generating bloated JSON schemas

Reply:
it was done automatically by running generateSchema not my intended change